docs/installationguide/ubuntu.rst

```diff
@@ -1,18 +1,99 @@
 Ubuntu
 ======
 
+
 PPA Install
 -----------
 Follow these instructions to install current release version on Ubuntu.:
 
+Pre-requisite
+^^^^^^^^^^^^^^^
+It is important that you first apply any system software upgrades first to Ubuntu, especially if you have just created a new image of Ubuntu.
+Not doing this may cause the PPA process to fail and complain about various unmet dependencies.
+
+
+If you also plan to install the database in the same server (which is typically the case), first do:
+
+::
+
+	sudo apt-get install mysql-server
+
+This will ask you for a user and password to configure for Zoneminder. 
+Note that when you install the PPA, it will also create a  username of zmuser and a password of zmpass irrespective of what you select at this stage
+
+
+Suggested changes to MySQL (Optional but recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For most of you Zoneminder will run just fine with the default MySQL settings. There are a couple of settings that may, in time, provide beneficial especially if you have a number of cameras and many events with a lot of files. One setting we recommend is the "innodb_file_per_table" This will be a default setting in MySQL 5.6 but should be added in MySQL 5.5 which comes with Ubuntu 14.04. A description can be found here: http://dev.mysql.com/doc/refman/5.5/en/innodb-multiple-tablespaces.html
+
+To add "innodb_file_per_table" edit the my.cnf file:
+
+``vi /etc/mysql/my.cnf``
+Under [mysqld] add
+``innodb_file_per_table``
+
+Save and exit.
+
+Restart MySQL
+``service mysql restart``
+
+Installing the actual Zoneminder package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+
   sudo apt-add-repository ppa:iconnor/zoneminder
 
 Once you have updated the repository then update and install the package.:
   
+::
+
   sudo apt-get update
   sudo apt-get install zoneminder
 
 
+Post Install Configuration
```

Comment:
Is this specific to Ubuntu? Could Post installation be a generic section to save repeating ourselves.